tests/ludwig/augmentation/test_augmentation_pipeline.py

```diff
@@ -139,14 +139,14 @@ def run_augmentation_training(
         },
     }
 
-    model = LudwigModel(config, logging_level=logging.INFO)
-    model.experiment(
-        dataset=train_fp,
-        skip_save_processed_input=True,
-        skip_save_model=True,
-    )
-    return model
-
+    with tempfile.TemporaryDirectory() as tmpdir:
+        model = LudwigModel(config, logging_level=logging.INFO)
+        model.experiment(
+            dataset=train_fp,
+            skip_save_processed_input=True,
+            skip_save_model=True,
+            output_directory=os.path.join(tmpdir, "output"),
+        )
```

Comment:
tmpdir fixture?

Reply:
Can the pytest fixture be used in a non-"test_" function?

Reply:
No. We would need to plumb it through, so would just add more complexity, IMO.

Reply:
Yup, I was thinking plumbing it through, but this is also ok for now